wagtail/contrib/redirects/models.py

```diff
@@ -27,6 +29,25 @@ class Redirect(models.Model):
         on_delete=models.CASCADE
     )
     redirect_link = models.URLField(verbose_name=_("redirect to any URL"), blank=True, max_length=255)
+    automatically_created = models.BooleanField(
+        verbose_name=_("automatically created"),
+        default=False,
+        editable=False,
+    )
+    created_at = models.DateTimeField(verbose_name=_("created at"), auto_now=True, null=True)
+    trigger_content_type = models.ForeignKey(
+        ContentType,
+        models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name='+',
+    )
+    trigger_id = models.PositiveIntegerField(
+        blank=True,
+        null=True,
+        db_index=True,
+    )
+    trigger = GenericForeignKey("trigger_content_type", "trigger_id")
```

Comment:
Why do we need to record this?

Reply:
I think this could potentially be very useful if you wanted to know more about why a particular redirect was created, and who actioned that.

Reply:
If this is useful to capture, could we capture it in a standard way? (Like an audit log, which can be created for any model now)

Reply:
I thought we'd established a while ago that we don't want to log every by-product of an action? this value is 'to-hand' and applying it to all items uses practically very few resources. Creating a separate log entry for every redirect is overkill, and simply redundant when that data is already logged.